presto-main/src/main/java/io/prestosql/server/ui/PasswordManagerFormAuthenticator.java

```diff
@@ -70,5 +72,9 @@ public boolean isValidCredential(String username, String password, boolean secur
         catch (AccessDeniedException e) {
             return false;
         }
+        catch (RuntimeException e) {
+            log.warn(e, "Error authenticating user for Web UI");
```

Comment:
would it help to add username in the error message for ease of debugging?

Reply:
If it is the way certain password authenticator works, then logging this on `warn` could easily pollute all server logs.

I would prefer to log on debug by default, then we could easily add more info there like @phd3 suggested.

BTW: do we need debug logging for above `AccessDeniedException`

Reply:
The assumption we have always used for Presto is you either log **or** throw.  So if we are catching an exception, we can assume it has not already been logged.